### ISSUE TITLE
skidded

### ISSUE THREAD

**Issue by Sadcrysbro** [0Zayn/Ballistic#6]
heres the original source: https://github.com/RbxStu/RbxStu-V2/
make your own shit

**Comment by 0Zayn** [comment]
Wtf do you mean "original source" are you okay????

This does not even have the same content, and it is my "own shit"

**Comment by Katrist** [comment]
@0Zayn 

Original Code: https://github.com/RbxStu/RbxStu-V2/blob/0cb2f80af99cf6f339e46bd1ce16254901f11d87/Environment/Libraries/Instance.cpp#L131
Pasted Code: https://github.com/0Zayn/Ballistic/blob/68fa737f0ae85f84074aa0952808eb6a6f72143f/Ballistic_Module/Source/Environment/Script/Script.cpp#L131

Original Code: https://github.com/RbxStu/RbxStu-V2/blob/dev/Roblox/TypeDefinitions.hpp
Pasted Code: https://github.com/0Zayn/Ballistic/blob/master/Ballistic_Module/Source/RBX/RBX.hpp

Original Code: https://github.com/RbxStu/RbxStu-V2/blob/0cb2f80af99cf6f339e46bd1ce16254901f11d87/Environment/Libraries/Metatable.cpp#L64
Pasted Code: https://github.com/0Zayn/Ballistic/blob/68fa737f0ae85f84074aa0952808eb6a6f72143f/Ballistic_Module/Source/Environment/Metatable/Metatable.cpp#L81

**Comment by 0Zayn** [comment]
> @0Zayn 
> 
> 
> 
> Original Code: https://github.com/RbxStu/RbxStu-V2/blob/0cb2f80af99cf6f339e46bd1ce16254901f11d87/Environment/Libraries/Instance.cpp#L131
> 
> Pasted Code: https://github.com/0Zayn/Ballistic/blob/68fa737f0ae85f84074aa0952808eb6a6f72143f/Ballistic_Module/Source/Environment/Script/Script.cpp#L131
> 
> 
> 
> Original Code: https://github.com/RbxStu/RbxStu-V2/blob/dev/Roblox/TypeDefinitions.hpp
> 
> Pasted Code: https://github.com/0Zayn/Ballistic/blob/master/Ballistic_Module/Source/RBX/RBX.hpp
> 
> 
> 
> Original Code: https://github.com/RbxStu/RbxStu-V2/blob/0cb2f80af99cf6f339e46bd1ce16254901f11d87/Environment/Libraries/Metatable.cpp#L64
> 
> Pasted Code: https://github.com/0Zayn/Ballistic/blob/68fa737f0ae85f84074aa0952808eb6a6f72143f/Ballistic_Module/Source/Environment/Metatable/Metatable.cpp#L81 
> 
> 
> 
> No hate, just providing some information

Yes I'm aware of those as of yesterday, but I wasn't the one who exactly commit those to the repo as I explained to many people. There are multiple people who worked on ballistic and I don't pay attention to stuff like that

I already had a talk with Dottik

However I don't paste, and I've never had a reputation for it

**Comment by ForlornWindow46** [comment]
just asking @0Zayn does this still work? as of today and is the offsets the reason its crashing?

**Comment by 0Zayn** [comment]
> just asking @0Zayn does this still work? as of today and is the offsets the reason its crashing?

You need to update everything in the "Update" folder its not all caught up with the newest Roblox version, that is why it crashes
